types/index.d.ts

```diff
@@ -7,80 +7,188 @@
 import * as stream from 'stream';
 import { EventEmitter } from 'events'
 
-type Schema = string | object;  // TODO object should be further specified
+//"virtual" namespace (no JS, just types) for Avro Schema
+declare namespace schema
+{
```

Comment:
Can you put the opening brace on the same line as the declaration for consistency with the rest of the file? (Throughout too.)

Reply:
ah, styling. Will do.